test/index.test.js

```diff
@@ -122,10 +131,14 @@ describe('Index Tests', () => {
         if (!fse.existsSync(path)) {
           return [404, `File not found: ${path}`];
         }
-        return [200, fse.readFileSync(path, 'utf-8'), {
-          'last-modified': 'Mon, 22 Feb 2021 15:28:00 GMT',
-          server: 'nock',
-        }];
+        const metadata = `${path}.json`;
+        let status = 200;
+        let headers = { 'last-modified': 'Mon, 22 Feb 2021 15:28:00 GMT' };
+
+        if (fse.existsSync(metadata)) {
+          ({ status, headers } = fse.readJSONSync(metadata));
+        }
+        return [status, fse.readFileSync(path, 'utf-8'), headers];
```

Comment:
When asking for a HTML, allow returning metadata as well in file with added extension `.json`